experimental/bootstrap/create.go

```diff
@@ -447,31 +447,41 @@ func InstanceTags(resourceTag ec2.Tag, gid group.ID) map[string]string {
 	}
 }
 
-const prepareGroupWatches = `
+const startInfraKit = `
```

Comment:
I suggest you push this into `initializeManager` so we only have one template to compose the manager boot script.